### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by artsiomkorzun**
#### Check List:
- [x] Tests pass (`./test.sh <username>` shows no differences between expected and actual outputs)
- [x] All formatting changes by the build are committed
- [x] Your launch script is named `calculate_average_<username>.sh` (make sure to match casing of your GH user name) and is executable
- [x] Output matches that of `calculate_average_baseline.sh`
- [x] For new entries, or after substantial changes: When implementing custom hash structures, please point to where you deal with hash collisions (line number)

* Execution time: ~30 ms better
* Execution time of reference implementation: a lot

### CODE REVIEW

File: src/main/java/dev/morling/onebrc/CalculateAverage_artsiomkorzun.java [hunk 1]
```diff
@@ -218,11 +227,10 @@ public static void update(long address, int value) {
             UNSAFE.putInt(address + 16, cnt);
 
             if (value < min) {
-                UNSAFE.putShort(address + 20, (short) value);
+                UNSAFE.putShort(address + 20, value);
             }
-
-            if (value > max) {
-                UNSAFE.putShort(address + 22, (short) value);
+            else if (value > max) {
```
Comment (royvanrijn) [inline src/main/java/dev/morling/onebrc/CalculateAverage_artsiomkorzun.java:232]: Nice, I guess it did make a difference? You could well be breaking 2 seconds now!

Comment (artsiomkorzun) [reply src/main/java/dev/morling/onebrc/CalculateAverage_artsiomkorzun.java:232]: A very small difference, which I can't confirm with 100% certainty. Not filling memory with zeros gives around 10ms as well. Lets see how it works out on the eval machine.

**Comment by gunnarmorling** [comment]
Comes in a bit slower actually:

```
Benchmark 1: timeout -v 300 ./calculate_average_artsiomkorzun.sh 2>&1
  Time (mean ± σ):      2.060 s ±  0.010 s    [User: 0.002 s, System: 0.004 s]
  Range (min … max):    2.034 s …  2.066 s    10 runs

Summary
  artsiomkorzun: trimmed mean 2.0625630242949997, raw times 2.0336724294199997,2.06647070542,2.06597330042,2.0661385814199997,2.05878286342,2.06480267942,2.0643172384199997,2.0615065864199997,2.0622090584199997,2.05677388642

Leaderboard

| # | Result (m:s.ms) | Implementation     | JDK | Submitter     | Notes     |
|---|-----------------|--------------------|-----|---------------|-----------|
|   | 00:02.062 | [link](https://github.com/gunnarmorling/1brc/blob/main/src/main/java/dev/morling/onebrc/CalculateAverage_artsiomkorzun.java)| 21.0.2-graal | [Artsiom Korzun](https://github.com/artsiomkorzun) | GraalVM native binary, uses Unsafe |
```

**Comment by artsiomkorzun** [comment]
@gunnarmorling reverted some changes. Please try once more.

**Comment by gunnarmorling** [comment]
Almost the same now as the current entry:

```
Benchmark 1: timeout -v 300 ./calculate_average_artsiomkorzun.sh 2>&1
  Time (mean ± σ):      2.022 s ±  0.015 s    [User: 0.001 s, System: 0.004 s]
  Range (min … max):    1.996 s …  2.036 s    5 runs

Summary
  artsiomkorzun: trimmed mean 2.0264423235666666, raw times 1.9956557839,2.0358313169000004,2.0279398379,2.0246280749000003,2.0267590579

Leaderboard

| # | Result (m:s.ms) | Implementation     | JDK | Submitter     | Notes     |
|---|-----------------|--------------------|-----|---------------|-----------|
|   | 00:02.026 | [link](https://github.com/gunnarmorling/1brc/blob/main/src/main/java/dev/morling/onebrc/CalculateAverage_artsiomkorzun.java)| 21.0.2-graal | [Artsiom Korzun](https://github.com/artsiomkorzun) | GraalVM native binary, uses Unsafe |
```

**Pull request closed without merging**